arangosh/Benchmark/BenchFeature.cpp

```diff
@@ -482,10 +526,13 @@ void BenchFeature::start() {
   *_result = ret;
 }
 
-bool BenchFeature::report(ClientFeature& client, std::vector<BenchRunResult> results,
+void BenchFeature::report(ClientFeature& client, std::vector<BenchRunResult>& results,
```

Comment:
I'd be in favor of passing `std::vector<BenchRunResult>& results` as `std::vector<BenchRunResult> const& results` and sort results in a caller. In `BenchFeature::report` we can add an assertion that results are sorted, like `TRI_ASSERT(std::is_sorted(...))`

Reply:
Addressed in latest commit, but can still have room for more optimization